### PR TITLE
Updated link to Visual Studio SDK docs

### PR DESCRIPTION
**Pull request by mgoertz-msft**

### CODE REVIEW

**Comment by PRMerger14** [comment]
@mgoertz-msft : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: docs/modeling/getting-started-with-domain-specific-languages.md [hunk 1]
```diff
@@ -39,8 +39,8 @@ To define a DSL, you must have installed the following components:
 
 | Component | Link |
 |-|-|
-| Visual Studio | [http://go.microsoft.com/fwlink/?LinkId=185579](https://visualstudio.microsoft.com/) |
-| [!INCLUDE[vssdk_current_short](../modeling/includes/vssdk_current_short_md.md)] | [http://go.microsoft.com/fwlink/?LinkId=185580](/azure/devops/integrate/index) |
+| Visual Studio | [http://go.microsoft.com/fwlink/?LinkId=185579](https://visualstudio.microsoft.com) |
+| [!INCLUDE[vssdk_current_short](../modeling/includes/vssdk_current_short_md.md)] | [https://go.microsoft.com/fwlink/?linkid=2166172](https://docs.microsoft.com/en-us/visualstudio/extensibility/visual-studio-sdk) |
```
Comment (v-andreaco) [inline docs/modeling/getting-started-with-domain-specific-languages.md:43]:
```suggestion
| [!INCLUDE[vssdk_current_short](../modeling/includes/vssdk_current_short_md.md)] | [https://go.microsoft.com/fwlink/?linkid=2166172](https://docs.microsoft.com/visualstudio/extensibility/visual-studio-sdk) |
```

**Comment by PRMerger6** [comment]
@v-andreaco : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger7** [comment]
@ktoliver : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by ktoliver** [comment]
Merging per email from @j-martens

**Pull request merged**